src/SigningRequest.php

```diff
@@ -206,4 +208,17 @@ public function setEnableInsecureSigning($enableInsecureSigning)
     {
         $this->enableInsecureSigning = $enableInsecureSigning;
     }
+
+    public function getInsecureSigningMethods(): ?array
+    {
+        return $this->insecureSigningMethods;
+    }
+
+    /**
+     * @param array<string>|null $insecureSigningMethods
+     */
+    public function setInsecureSigningMethods(?array $insecureSigningMethods): void
```

Comment:
The actual type is `array<"image" | "text" | "draw">` or whatever.

Reply:
Fair. I could update to php 8.1 and have actual enums but I feel that's outside of the scope of this PR.